sdks/python/apache_beam/portability/api/standard_coders.yaml

```diff
@@ -0,0 +1,410 @@
+#
```

Comment:
We shouldn't add this, it should be copied on demand. Otherwise things may diverge which defeats the purpose of standard coders.

Reply:
Deleted.